api/cfnetworking/cfnetv1/policy_test.go

```diff
@@ -114,4 +111,188 @@ var _ = Describe("Policy", func() {
 			})
 		})
 	})
+
+	Describe("ListPolicies", func() {
+		var expectedPolicies []Policy
+		Context("when the stack is found", func() {
+			BeforeEach(func() {
+				expectedResponse := `{
+					"policies": [
+						{
+							"source": {
+								"id": "source-id-1"
+							},
+							"destination": {
+								"id": "destination-id-1",
+								"protocol": "tcp",
+								"ports": {
+									"start": 1234,
+									"end": 1235
+								}
+							}
+						}
+					]
+				}`
+				server.AppendHandlers(
+					CombineHandlers(
+						VerifyRequest(http.MethodGet, "/policies"),
+						RespondWith(http.StatusOK, expectedResponse),
+					),
+				)
+				expectedPolicies = []Policy{
+					{
+						Source: PolicySource{
+							ID: "source-id-1",
+						},
+						Destination: PolicyDestination{
+							ID:       "destination-id-1",
+							Protocol: "tcp",
+							Ports: Ports{
+								Start: 1234,
+								End:   1235,
+							},
+						},
+					},
+				}
+			})
+
+			It("returns the policies correctly", func() {
+				policies, err := client.ListPolicies("")
```

Comment:
This and line 193 would no longer have to pass empty string to `ListPolicies` when it becomes variadic.